From e785a4e180259aed351d8d94eaf8f5b11ae0cbb9 Mon Sep 17 00:00:00 2001
From: Paul Maunders <paul@fubra.com>
Date: Thu, 19 Dec 2013 16:04:06 +0100
Subject: [PATCH 01/10] Fixed a small typo in the man page

and regenerate the man pages.

(cherry picked from commit cf874f1b67fd4cb89486d1208baf8225ede38cd7)
Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1024151
Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

Conflicts:
	AUTHORS
---
 AUTHORS                    |  1 +
 man/en/virt-install.pod    | 32 +++++++++++++++++++++-----------
 man/en/virt-install.pod.in |  2 +-
 3 files changed, 23 insertions(+), 12 deletions(-)

diff --git a/AUTHORS b/AUTHORS
index 7f6477c..d08de48 100644
--- a/AUTHORS
+++ b/AUTHORS
@@ -52,6 +52,7 @@ Patches also received from
     Andres Rodriguez <andreserl-at-ubuntu-dot-com>
     Hajime Taira <htaira-at-redhat-dot-com>
     Martin Kletzander <mkletzan-at-redhat-dot-com>
+    Paul Maunders <paul-at-fubra-dot-com>
 
     ...send patches and get your name here...
 
diff --git a/man/en/virt-install.pod b/man/en/virt-install.pod
index b3ec9e6..a0b2efb 100644
--- a/man/en/virt-install.pod
+++ b/man/en/virt-install.pod
@@ -341,20 +341,24 @@ Values for some recent OS options are:
 
 =item generic              : Generic
 
+=item debianwheezy         : Debian Wheezy
+
 =item debiansqueeze        : Debian Squeeze
 
 =item debianlenny          : Debian Lenny
 
+=item fedora18             : Fedora 18
+
+=item fedora17             : Fedora 17
+
 =item fedora16             : Fedora 16
 
 =item fedora15             : Fedora 15
 
-=item fedora14             : Fedora 14
+=item mageia1              : Mageia 1 and later
 
 =item mes5.1               : Mandriva Enterprise Server 5.1 and later
 
-=item mandriva2010         : Mandriva Linux 2010 and later
-
 =item rhel6                : Red Hat Enterprise Linux 6
 
 =item rhel5.4              : Red Hat Enterprise Linux 5.4 or later
@@ -365,13 +369,19 @@ Values for some recent OS options are:
 
 =item sles10               : Suse Linux Enterprise Server
 
+=item opensuse12           : openSuse 12
+
+=item opensuse11           : openSuse 11
+
+=item ubuntuquantal        : Ubuntu 12.10 (Quantal Quetzal)
+
+=item ubuntuprecise        : Ubuntu 12.04 LTS (Precise Pangolin)
+
 =item ubuntuoneiric        : Ubuntu 11.10 (Oneiric Ocelot)
 
 =item ubuntunatty          : Ubuntu 11.04 (Natty Narwhal)
 
-=item ubuntumaverick       : Ubuntu 10.10 (Maverick Meerkat)
-
-=item ubuntulucid          : Ubuntu 10.04 (Lucid Lynx)
+=item ubuntulucid          : Ubuntu 10.04 LTS (Lucid Lynx)
 
 =item ubuntuhardy          : Ubuntu 8.04 LTS (Hardy Heron)
 
@@ -486,10 +496,11 @@ size (in GB) to use if creating new storage
 =item B<sparse>
 
 whether to skip fully allocating newly created storage. Value is 'true' or
-'false'. Default is 'true' (do not fully allocate).
+'false'. Default is 'true' (do not fully allocate) unless it isn't
+supported by the underlying storage type.
 
 The initial time taken to fully-allocate the guest virtual disk (sparse=false)
-will be usually by balanced by faster install times inside the guest. Thus
+will be usually balanced by faster install times inside the guest. Thus
 use of this option is recommended to ensure consistently high performance
 and to avoid I/O errors in the guest should the host filesystem fill up.
 
@@ -508,7 +519,7 @@ L<http://libvirt.org/storage.html> for possible values. This is often
 mapped to the B<driver_type> value as well.
 
 With libvirt 0.8.3 and later, this option should be specified if reusing
-and existing disk image, since libvirt does not autodetect storage format
+an existing disk image, since libvirt does not autodetect storage format
 as it is a potential security issue. For example, if reusing an existing
 qcow2 image, you will want to specify format=qcow2, otherwise the hypervisor
 may not be able to read your disk image.
@@ -626,7 +637,7 @@ Connect to a virtual network in the host called C<NAME>. Virtual networks
 can be listed, created, deleted using the C<virsh> command line tool. In
 an unmodified install of C<libvirt> there is usually a virtual network
 with a name of C<default>. Use a virtual network if the host has dynamic
-networking (eg NetworkManager), or using wireless. The guest will be 
+networking (eg NetworkManager), or using wireless. The guest will be
 NATed to the LAN by whichever connection is active.
 
 =item user
@@ -1320,4 +1331,3 @@ is NO WARRANTY, to the extent permitted by law.
 C<virsh(1)>, C<virt-clone(1)>, C<virt-manager(1)>, the project website C<http://virt-manager.org>
 
 =cut
-
diff --git a/man/en/virt-install.pod.in b/man/en/virt-install.pod.in
index 3d2dee8..69a6bd3 100644
--- a/man/en/virt-install.pod.in
+++ b/man/en/virt-install.pod.in
@@ -455,7 +455,7 @@ L<http://libvirt.org/storage.html> for possible values. This is often
 mapped to the B<driver_type> value as well.
 
 With libvirt 0.8.3 and later, this option should be specified if reusing
-and existing disk image, since libvirt does not autodetect storage format
+an existing disk image, since libvirt does not autodetect storage format
 as it is a potential security issue. For example, if reusing an existing
 qcow2 image, you will want to specify format=qcow2, otherwise the hypervisor
 may not be able to read your disk image.
-- 
1.8.5.3

